### PR TITLE
Rewrote maps page

### PR DESCRIPTION
**Pull request by sandlerben**

### CODE REVIEW

File: app/forms.py [hunk 1]
```diff
@@ -318,3 +319,56 @@ def validate_name_plural(form, field):
...
+class CheckboxForm(Form):
+	id = TextField()
+	value = BooleanField(
+		validators = []
+	)
+
+	def set_label(self, label):
+		self.value.label = Label(self.value.id, label)
+
+class PrecheckedCheckboxForm(Form):
```
Comment (weronica) [inline app/forms.py]: This model seems redundant. What do you think about inheriting from `CheckboxForm`? Or overriding `CheckboxForm`'s `__init__` method to take a boolean to set the default value for the `BooleanField`?

File: app/static/css/styles.css [hunk 2]
```diff
@@ -896,3 +896,8 @@ li {
   margin-bottom: 5px;
   padding: 8px 10px;
 }
+
+#size-limit-announcement {
+  font-weight: bold;
+  color: #ef3d23;
```
Comment (weronica) [inline app/static/css/styles.css]: Could you include a comment that identifies what color this is?

File: app/templates/base.html [hunk 3]
```diff
@@ -142,7 +142,7 @@ <h4 id="hr-helper">.</h4>
       </div>
       <div class="small-2 columns right">
         <ul id="footer-pics">
-          <li class="footer-li"> <a href="http://hack4impact.weebly.com/"><img class="footer-img" src="{{ url_for('static', filename='img/h4i_logo.png') }}" alt="Hack4Impact"/></a></li>
+          <li class="footer-li"> <a href="http://hack4impact.com/"><img class="footer-img" src="{{ url_for('static', filename='img/h4i_logo.png') }}" alt="Hack4Impact"/></a></li>
```
Comment (weronica) [inline app/templates/base.html]: The new website's URL is actually hack4impact.org.

File: app/forms.py (hunk 1 above)
Comment (sandlerben) [inline app/forms.py]: Sure.

**Pull request merged**